src/libs/ReportUtils.js

```diff
@@ -702,8 +702,7 @@ function buildOptimisticIOUReport(ownerEmail, userEmail, total, chatReportID, cu
  */
 function getIOUReportActionMessage(type, total, participants, comment, currency) {
     const amount = NumberFormatUtils.format(preferredLocale, total / 100, {style: 'currency', currency});
-    const isMultipleParticipantReport = participants.length > 1;
-    const displayNames = _.map(participants, participant => getDisplayNameForParticipant(participant.login, isMultipleParticipantReport) || participant.login);
+    const displayNames = _.map(participants, participant => getDisplayNameForParticipant(allPersonalDetails[participant.login], true) || participant.login);
```

Comment:
This is a throwing a `text.replace` is not function for me. Due to the change [here](https://github.com/Expensify/App/pull/12399/files#diff-65c096044d5f69b35bcdec14c99c4fda4580759df9b1a7c36650d58eea276f1d), `getDisplayNameForParticipant ` now expects a string instead of the `participant` object.

Reply:
Oh, you are completely right! 

Fixed and tested. Seems to be working!